src/Components/App.lua

```diff
@@ -12,9 +12,21 @@ local loader = ModuleLoader.new()
 local function App(_props, hooks: any)
 	local theme = useTheme(hooks)
 	local storybooks = useStorybooks(hooks, game, loader)
-	local story, selectStory = hooks.useState(nil)
+	local story, setStory = hooks.useState(nil)
 	local storybook, selectStorybook = hooks.useState(nil)
 
+	local selectStory = hooks.useCallback(function(newStory: ModuleScript)
+		setStory(function(prevStory)
+			if prevStory then
+				if prevStory == newStory then
+					return nil
+				end
+			end
+
+			return newStory
+		end)
```

Comment:
Would this behave the same?
```suggestion
		setStory(function(prevStory: types.Story)
			return if prevStory ~= newStory then newStory else nil
		end)
```